### ISSUE TITLE
Uncaught TypeError: onClickOutside is not a function

### ISSUE THREAD

**Issue by NeverMore93** [YouCanBookMe/react-datetime#395]
```
Uncaught TypeError: onClickOutside is not a function
    at eval (webpack:///./~/react-datetime/src/DaysView.js?:9)
    at Object.<anonymous> (index.js:13029)
    at __webpack_require__ (index.js:585)
    at fn (index.js:109)
    at eval (webpack:///./~/react-datetime/src/CalendarContainer.js?:5)
    at Object.<anonymous> (index.js:13023)
    at __webpack_require__ (index.js:585)
    at fn (index.js:109)
    at eval (webpack:///./~/react-datetime/DateTime.js?:8)
    at Object.<anonymous> (index.js:13011)
    at __webpack_require__ (index.js:585)
    at fn (index.js:109)
    at Object.eval (webpack:///./src/js/components/NewSAM/ScheduleLayer.js?:69)
    at Object.eval (webpack:///./src/js/components/NewSAM/ScheduleLayer.js?:435)
    at eval (webpack:///./src/js/components/NewSAM/ScheduleLayer.js?:436)
    at Object.<anonymous> (index.js:12879)
```

**Comment by Slapbox** [comment]
Having the same issue.

**Comment by Amonith** [comment]
+1 The lib is working fine when used directly in one of my projects, but when I create a npm lib for my common components (one of which uses react-datetime) and link this lib using npm link to some other app, this is the error I get.

My lib doesn't really use anything weird. The components were moved from the main project.
If I remove the component which uses react-datetime everything works fine.

**Comment by Amonith** [comment]
Ok, I've found out that this is because react-onlickoutside above version 5.x is incompatible with this lib.
@NeverMore93 you may wish to downgrade your dependency on this.

**Comment by jaselv** [comment]
The breaking change that was made to react-onclickoutside is that you have to do require('react-onclickoutside').default instead of require('react-onclickoutside'). I tested this with react-onclickoutside@6.5.0

**Comment by erquhart** [comment]
This is addressed in 2.11.0 via #434.
